src/main/java/services/task_presentation/TodoListInfoFromTaskReaders.java

```diff
@@ -31,4 +31,8 @@ public TodoListInfoFromTaskReaders(Map<Long, List<TaskReader>> taskReaders) {
             taskInfos.add(new TaskInfoFromTaskReader(tr));
         return taskInfos;
     }
+
+    public List<TaskReader> getTaskReaders() {
```

Comment:
If you want to expose the Readers to the adapter and outward layers, might as well deprecate the -Info classes, but make that kind of change another PR. I think I was going to eventually go about doing this with the observable repositories PR but I hadn't committed to the idea yet

Reply:
Ok I removed the getter